.github/workflows/release_pypi.yml

```diff
@@ -0,0 +1,104 @@
+name: PyPI Release
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This needs

```
with:
  fetch-depth: 0
```

in order to avoid a wonky run-time bug. tl;dr PyOxidizer's build script looks at the Git history of its checkout to bake some settings into the built binary. It gets confused when running from shallow clones, which GitHub Actions does by default.

This change needs to be made throughout the file.